frontend/src/components/Registration/Registration.tsx

```diff
@@ -82,17 +98,32 @@ function Registration() {
                 </Typography>
                 <Snackbar
                   anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-                  open={visibleSucces}
+                  sx={{ zIndex: 10000 }}
+                  open={!!notification}
+                  autoHideDuration={3000}
+                  onClose={handleCloseNotification}
                 >
-                  <Alert severity="success">
-                    {t('registration.regisrationSuccess')}
+                  <Alert
+                    onClose={handleCloseNotification}
+                    severity="error"
+                    sx={{ mt: '1vh' }}
+                  >
+                    {notification}
                   </Alert>
                 </Snackbar>
                 <Snackbar
                   anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-                  open={!!errorMessage}
+                  sx={{ zIndex: 10000 }}
+                  open={!!visibleSucces}
+                  onClose={handleCloseNotification}
                 >
-                  <Alert severity="error">{errorMessage}</Alert>
+                  <Alert
+                    onClose={handleCloseNotification}
+                    severity="success"
+                    sx={{ mt: '1vh' }}
+                  >
+                    {t('registration.regisrationSuccess')}
+                  </Alert>
                 </Snackbar>
                 <Controller

```

Comment:
<img width="1536" alt="Screenshot 2022-05-30 at 14 02 49" src="https://user-images.githubusercontent.com/39120801/170979196-36aaf26a-c14d-48f5-8a8b-9cf4a7da22a7.png">
should correct this problem